docs/_attic/glossary_entries.py

```diff
@@ -241,6 +254,13 @@
 	institute="Dockstore", 
 	pronunciation='')
 
+EnvironmentVariable = GlossEntry("environment variable", 
```

Comment:
I wonder if this is getting a little too low-level. We have to assume some knowledge, and, for example, I think the sentence you have below, "You can set environment variables in Bash using the export command" while not wrong, also isn't precise:

* It's not a Bash-only feature
* Using export isn't the only way to set them.

But more broadly, I don't think we should be discussing different shells, the difference between SET and EXPORT, etc. in the Dockstore documentation.

Just my opinion, feel free to push back.

Reply:
Almost all of our documentation assumes a bash-on-Ubuntu environment (it's just that bash-on-Mac and zsh-on-Mac are 99% the same), so I think using that as an example of an env variable is the best example. Good point about set and export though, I'll see what I can think of...